### PR TITLE
docs(MessageManager): add clarification to fetch messages

### PR DESCRIPTION
**Pull request by andre-paulo98**
**Please describe the changes this PR makes and why it should be merged:** [Discord's documentation](https://discord.com/developers/docs/resources/channel#get-channel-messages) mentions that you can only use `before`, `after` or `around` when fetching messages. This is not mentioned in D.JS documentation, and after some time trying to figure out why I was getting more messages than I should, decided to read the documentation.
This only changes the documentation.

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are **no code changes**
- I know how to update typings and have done so, **or typings don't need updating**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

*I submitted this before on #9220 but I messed up with something on my side for the second commit*

### CODE REVIEW

File: packages/discord.js/src/managers/MessageManager.js [hunk 1]
```diff
@@ -49,6 +49,8 @@ class MessageManager extends CachedManager {
 
   /**
    * Options used to fetch multiple messages.
+   * <info>The `before`, `after`, and `around` parameters are mutually exclusive,
```
Comment (Jiralite) [inline packages/discord.js/src/managers/MessageManager.js:52]:
```suggestion
   * <info>The `before`, `after`, and `around` parameters are mutually exclusive.
```

**Review by Jiralite — APPROVED** [review]

File: packages/discord.js/src/managers/MessageManager.js [hunk 2]
```diff
@@ -49,6 +49,8 @@ class MessageManager extends CachedManager {
...
+   * only one may be passed at a time.</info>
```
Comment (jaw0r3k) [inline packages/discord.js/src/managers/MessageManager.js:53]: Also (mutually exclusive)[https://www.collinsdictionary.com/dictionary/english/mutually-exclusive] is self explaining,
so no need for this line

Comment (andre-paulo98) [reply packages/discord.js/src/managers/MessageManager.js:53]: I used the same text they have on Discord Documentation: https://discord.com/developers/docs/resources/channel#get-channel-messages

Comment (andre-paulo98) [inline packages/discord.js/src/managers/MessageManager.js:53]:
```suggestion
   * Only one may be passed at a time.</info>
```

**Review by andre-paulo98 — COMMENTED** [review]
if we're changing the colon to point ([@Jiralite suggestion](https://github.com/discordjs/discord.js/pull/9222#pullrequestreview-1335898002)), we'll also need to change this

File: packages/discord.js/src/managers/MessageManager.js (hunk 2 above)
Comment (jaw0r3k) [reply packages/discord.js/src/managers/MessageManager.js:53]: Like below djs only mentions that it is mutually exclusive
https://github.com/discordjs/discord.js/blob/1b29099ed0b0deb98db844671aa23b4a84ec9c08/packages/discord.js/src/managers/ThreadManager.js#L78

Comment (andre-paulo98) [reply packages/discord.js/src/managers/MessageManager.js]: updated it

**Review by vladfrangu — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**